pallets/micropayment/src/tests.rs

```diff
@@ -75,11 +73,11 @@ fn open_channel() {
         {
             assert_eq!(
                 dispatch_error_with_post_info.error,
-                DispatchError::Module {
+                DispatchError::Module(ModuleError {
```

Comment:
Since we have explicitly specified module index in `runtime/src/lib.rs`, I think we can also specify it in `pallets/micropayment/src/mock.rs`, then we can better understand what `index: 4` means.

Reply:
only runtime/src/lib.rs need spec index, pallets/micropayment/src/mock.rs do not need.